deployments/examples/ocis_traefik/docker-compose.yml

```diff
@@ -41,32 +42,13 @@ services:
 
   ocis:
     image: owncloud/ocis:${OCIS_DOCKER_TAG:-latest}
-    entrypoint:
-      - /bin/sh
-      - -c
-      - | # as long as https://github.com/owncloud/product/issues/15 is open we need this step to template konnectd config
-        cp /config/identifier-registration.dist.yaml /config/identifier-registration.yaml
-        sed -i 's/ocis.owncloud.test/${OCIS_DOMAIN:-ocis.owncloud.test}/g' /config/identifier-registration.yaml
-        ocis server
     networks:
       default:
     environment:
-      # general config
       OCIS_URL: https://${OCIS_DOMAIN:-ocis.owncloud.test}
-      OCIS_LOG_LEVEL: ${OCIS_LOG_LEVEL:-error}
-      # proxy config
-      PROXY_OIDC_INSECURE: "${INSECURE:-false}"
-      PROXY_TLS: "false"
```

Comment:
so you want to use tls encrypted traffic between traefik and the proxy? just because it is less config? AFAICT it is either `PROXY_TLS: "false"` or `- "--serversTransport.insecureSkipVerify=true" # oCIS uses self generated certificate` above.
Granted, SSL offloading is another topic ... might be worth a section in the example doc?

Reply:
I went for the default which is ssl enabled, so that we can remove this too: 
https://github.com/owncloud/ocis/blob/eb2143d3ece018d218c8e9351e0b82ad0da4360f/deployments/examples/ocis_keycloak/docker-compose.yml#L65